### ISSUE TITLE
Equicord & Vencord Spellchecker not Working

### ISSUE THREAD

**Issue by refurbishing** [smartfrigde/armcord#766]
**Describe the bug**
Equicord & Vencord Spellchecker doesn't work 

**To Reproduce**
1. Go to the Chat Bar
2. Type a word incorrectly, select and right click on it
3. See No Dictionary, Spell Checker suggestions & Settings, 

**Expected behavior**
The Spell Checker working correctly.

**Screenshots**
* Working SpellChecker (ex: Equibop)
![image](https://github.com/user-attachments/assets/bfce4a67-ee58-4180-a4e7-4b9d9d678e17)

* Not Working SpellChecker (Legcord):
![image](https://github.com/user-attachments/assets/bba7f472-d5d6-4d56-99db-8367b4d5959c)

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Method of installation: AUR
 - Version: 1.0.2

**Additional context**
Someone already reported this on the [dev-build-logs](https://discord.com/channels/820732039253852171/1290857660584955904) channel over a month ago. The Thread OP only specified Equicord but this happens on Vencord aswell.

**Comment by samjaninf** [comment]
I have the same issue with Equicord.

**Comment by Valiant-0** [comment]
Oh hello, just to confirm it is not working.

**Comment by alorgill** [comment]
+1

**Comment by jenh24** [comment]
I have the same issue.
Win 11 running Equicord

**Comment by Wheels35** [comment]
+1 for not working in Equicord, worked in Vencord
Windows 11 24h2, fwiw

**Comment by digitalbarrito** [comment]
Same issue, latest version of legcord (at time of writing) on macOS (tahoe dev beta fwiw), vencord spellcheck works, equicord does not.

**Comment by falchion10** [comment]
I'm on the latest version as well with macOS 26.0 Developer Beta 1 and having this issue. It doesn't seem like it's an issue specific to Windows or macOS though.

**Comment by refurbishing** [comment]
This issue seems to be solved on Vencord

**Comment by digitalbarrito** [comment]
> This issue seems to be solved on Vencord

Correct, spellcheck works with Vencord, still broken with Equicord via Legcord. 

Vencord has a different issue with Legcord where the QuickCSS window doesn't render correctly (still works you just can't see anything you're doing in the editor). 

So it's pick your poison currently. Vencord seems a bit more stable/bug free in general than Equicord right now with Legcord.

**Comment by thororen1234** [comment]
![Image](https://github.com/user-attachments/assets/84b0d548-10b7-414d-b7c1-b45874cc7f9f)

disable add back and spellcheck will work

I just made it so the default will be off on legcord you will still need to update your settings though

**Comment by thororen1234** [comment]
![Image](https://github.com/user-attachments/assets/6b51bfd8-0926-42e7-9ade-eb0dd6eb0609)
with add back off in vc & ec ^
![Image](https://github.com/user-attachments/assets/2e16aee9-609a-4ec7-a20f-9019692a8fab)
with add back on in vc & ec ^

**Comment by digitalbarrito** [comment]
> ![Image](https://github.com/user-attachments/assets/84b0d548-10b7-414d-b7c1-b45874cc7f9f)
> 
> disable add back and spellcheck will work
> 
> I just made it so the default will be off on legcord you will still need to update your settings though

Can confirm, this fixes spellcheck for me.

**Comment by AlterNezKo** [comment]
> ![Image](https://github.com/user-attachments/assets/84b0d548-10b7-414d-b7c1-b45874cc7f9f)
> 
> disable add back and spellcheck will work
> 
> I just made it so the default will be off on legcord you will still need to update your settings though

![Image](https://github.com/user-attachments/assets/d03f2c75-7940-412a-89de-0e9b53d7c72e)

In my case, it does not allow me to disable this option, as it claims to be necessary for the app to function properly.

![Image](https://github.com/user-attachments/assets/1d75c7b6-a6a2-4278-887d-dda5b9683da8)

**Comment by HTTP-Requests** [comment]
> > ![Image](https://github.com/user-attachments/assets/84b0d548-10b7-414d-b7c1-b45874cc7f9f)
> > disable add back and spellcheck will work
> > I just made it so the default will be off on legcord you will still need to update your settings though
> 
> ![Image](https://github.com/user-attachments/assets/d03f2c75-7940-412a-89de-0e9b53d7c72e)
> 
> In my case, it does not allow me to disable this option, as it claims to be necessary for the app to function properly.
> 
> ![Image](https://github.com/user-attachments/assets/1d75c7b6-a6a2-4278-887d-dda5b9683da8)

same here, resetting it does not work either.

**Comment by KrstlSkll69** [comment]
> same here, resetting it does not work either.

@HTTP-Requests @AlterNezKo 
Are you using Equicord or Vencord?

**Comment by Wheels35** [comment]
As a 'fix' for me, I am running as Equicord. Since I cant disable WebContextMenus with that toggle, If i click the cog, I can toggle it off but then it will tell me It needs to be on, restart now or later. I said later, manually restarted (clicked on system tray restart) and when it came back up I checked settings, saw that it was still 'enabled' but the cog still had it disabled.

Right-click now has the corrections, but does lose the other functionality, like custom searches.
